src/style.css

```diff
@@ -0,0 +1,19 @@
+.block {
+  height: 100px;
+}
+
+.red {
+  background-color: red;
+}
+
+.green {
+  background-color: green;
+}
+
+.blue {
+  background-color: blue;
+}
+
+* {
```

Comment:
Do not use the asterisk selector(`*`) without necessary. It could be used only for `box-sizing: border-box;`